welcome/learn_more_about_openshift.adoc

```diff
@@ -98,7 +98,7 @@ Use the following sections to find content to help you learn about and use {prod
 |
 
 |
-| xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Monitoring]
+| xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Monitoring stack]
```

Comment:
Looking at the other items in this column, "Monitoring" seems consistent and it is more than just the monitoring stack.